### PR TITLE
Issue-1-create-an-entry | Cristofer Martinez

### PR DESCRIPTION
**Pull request by CristoferMar**
issue-1-create-an-entry

### CODE REVIEW

**Comment by CristoferMar** [comment]
image

File: js/main.js [hunk 1]
```diff
@@ -1,2 +1,46 @@
 /* global data */
 /* exported data */
+
+var $photoURL = document.querySelector('.photoURL');
+var $newEntryImg = document.querySelector('.newEntryImg');
+
+// console.log($form.elements.target);
+// console.dir($form);
+// console.log($submit);
```
Comment (codingcodymiller) [inline js/main.js]: Please remove commented code before submitting 👍

File: js/main.js [hunk 2]
```diff
@@ -1,2 +1,46 @@
...
+});
+
+var $form = document.querySelector('.newEntry');
+
+var $submit = document.querySelector('.button');
+$submit.addEventListener('click', function (event) {
+  event.preventDefault();
+  var title = document.querySelector('.title').value;
+  var photoURL = document.querySelector('.photoURL').value;
+  var notes = document.querySelector('.notes').value;
```
Comment (codingcodymiller) [inline js/main.js]: You should use the `elements` property on the `form` element object instead of querying the DOM

File: js/main.js [hunk 3]
```diff
@@ -1,2 +1,46 @@
...
+  data.nextEntryId++;
+
+  $form.reset();
+  $newEntryImg.setAttribute('src', 'images/placeholder-image-square.jpg');
+
+  var dataJSON = JSON.stringify(data);
+  var entriesJSON = JSON.stringify(data.entries);
+
+  localStorage.setItem('data', dataJSON);
+  localStorage.setItem('entries', entriesJSON);
```
Comment (codingcodymiller) [inline js/main.js]: All localStorage work should be done in `data.js`, not `main.js`. Also, you don't need to save the entries separately, you are saving the entries by saving the `data` object

File: js/main.js [hunk 4]
```diff
@@ -1,2 +1,46 @@
...
+
+  // console.log('data.entries', data.entries);
+  // console.log('data.nextEntryId', data.nextEntryId);
```
Comment (codingcodymiller) [inline js/main.js]: Please remove commented code before submitting 👍

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
A few notes for you, great work so far!!

File: js/data.js [hunk 5]
```diff
@@ -6,3 +6,8 @@ var data = {
   editing: null,
   nextEntryId: 1
 };
+
```
Comment (codingcodymiller) [inline js/data.js:9]: You should add some code to check if there is a value in localStorage when the page loads, and if there is, replace the current value of `data` with that parsed object

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**